chromadb/test/property/test_cross_version_persist.py

```diff
@@ -233,7 +238,12 @@ def test_cycle_versions(
     ctx = multiprocessing.get_context("spawn")
     p = ctx.Process(
         target=persist_generated_data_with_old_version,
-        args=(version, settings, collection_strategy, embeddings_strategy),
+        args=(
+            version,
+            settings.persist_directory,
```

Comment:
Also changing this so we don't pass settings objects over process barrier. I think we should keep this change.